tests/Microsoft.Bot.Connector.Tests/GetTokenRefreshTests.cs

```diff
@@ -26,10 +26,10 @@ public async Task TokenTests_GetCredentialsWorks()
         }
 
 
-        //[Fact]
+        [Fact]
         public async Task TokenTests_RefreshTokenWorks()
         {
-            MicrosoftAppCredentials credentials = new MicrosoftAppCredentials("12604f0f-bc92-4318-a6dd-aed704445ba4", "H_k}}7b75BEl+KY1");
+            MicrosoftAppCredentials credentials = new MicrosoftAppCredentials("645cd89f-a83e-4af9-abb5-a454e917cbc4", "jvoMWRBA67:zjgePZ359_-_");
```

Comment:
Please move these passwords into encrypted variables in the build system.

Reply:
Use the "Secrets" function of Azure Devops for this:
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch